### DISCUSSION TITLE
My custom HTML for the Head is not displaying the image referenced in the img element.

### DISCUSSION THREAD

**Discussion by IO101010** [GetPublii/Publii#1872] [Help] [answered]
My custom HTML renders the header well on each page, except for the image. The element is set with src="/wp-content/uploads/oh-logo-round.png" and the wp-content page exists in the root directory (parallel to the input, output and preview directories). How come the image is not displayed in the preview?

**Comment by candidexmedia** [comment, marked as answer]
> the wp-content page exists in the root directory (parallel to the input, output and preview directories)

I could be wrong, but I don't think that Publii can fetch images stored in folders outside of input...

Your safest bet would be to host the image online (on something like Cloudinary, IMGUR, Backblaze), and link to that URL file.

See parallel discussions:
- https://github.com/GetPublii/Publii/discussions/1750#discussioncomment-10930728
- https://github.com/GetPublii/Publii/discussions/1751

**Comment by michaelfluegge** [comment]
for fonts I found this: https://getpublii.com/dev/dynamic-assets/
I think it is the same with other assets, meaning, they have to be stored in the assets directory, as @candidexmedia mentioned already

**Comment by IO101010** [comment]
Thanks for the direction. I wouldn't mind refactoring images site-wide if it was to a local directory such as assets. I tried putting it in /input/themes/assets but that it didn't work no matter how I tried linking to it.

Linking off site does work so I'll mark this as solved, but I don't like the idea of having to using absolute links to assets. I tend to make self-contained websites with relative links for portability. But I'll save that for a feature request. Thanks again!

**Comment by IO101010** [comment]
Thís problem seems to run a bit deeper. The links in my custom HTML also don't work. For example, the link to my About page is set with src="/about". When I preview the website it doesn't work and when I hover the mouse on the link it displays "file:///about".

**Reply by bttrx** [reply]
Preview mode uses only files from your local hard disk and I guess, there is no folder "about" in your root folder of your hard disk. It should work, if you sync/publish the site to your server.

**Reply by IO101010** [reply]
Thank you for that. It seems like a shortcoming of preview mode. I can work around it, but I doubt the people I want to hand Publii over to in the end can.
